lib/dependencies.sh

```diff
@@ -22,9 +22,9 @@ run_if_present() {
   local script_name=${2:-}
   local has_script
 
-  has_script=$(read_json "$build_dir/package.json" ".scripts[\"$script_name\"]")
+  has_script=$(has_script "$build_dir/package.json" "$script_name")
```

Comment:
Could you differentiate the names here to avoid conflating `has_script` the var with `has_script()` the fn?

Reply:
Good catch!